patches/@mantine+core+7.16.3.patch

```diff
@@ -0,0 +1,65 @@
+diff --git a/node_modules/@mantine/core/cjs/core/MantineProvider/MantineProvider.cjs b/node_modules/@mantine/core/cjs/core/MantineProvider/MantineProvider.cjs
+index d0006e5..49e2259 100644
+--- a/node_modules/@mantine/core/cjs/core/MantineProvider/MantineProvider.cjs
++++ b/node_modules/@mantine/core/cjs/core/MantineProvider/MantineProvider.cjs
+@@ -13,7 +13,7 @@ var suppressNextjsWarning = require('./suppress-nextjs-warning.cjs');
+ var useProviderColorScheme = require('./use-mantine-color-scheme/use-provider-color-scheme.cjs');
+ var useRespectReduceMotion = require('./use-respect-reduce-motion/use-respect-reduce-motion.cjs');
+ 
+-suppressNextjsWarning.suppressNextjsWarning();
++// suppressNextjsWarning.suppressNextjsWarning();
+ function MantineProvider({
+   theme,
+   children,
+diff --git a/node_modules/@mantine/core/esm/core/MantineProvider/MantineProvider.mjs b/node_modules/@mantine/core/esm/core/MantineProvider/MantineProvider.mjs
+index e928597..08a9360 100644
+--- a/node_modules/@mantine/core/esm/core/MantineProvider/MantineProvider.mjs
++++ b/node_modules/@mantine/core/esm/core/MantineProvider/MantineProvider.mjs
+@@ -11,7 +11,7 @@ import { suppressNextjsWarning } from './suppress-nextjs-warning.mjs';
+ import { useProviderColorScheme } from './use-mantine-color-scheme/use-provider-color-scheme.mjs';
+ import { useRespectReduceMotion } from './use-respect-reduce-motion/use-respect-reduce-motion.mjs';
+ 
+-suppressNextjsWarning();
++// suppressNextjsWarning();
+ function MantineProvider({
+   theme,
+   children,
+diff --git a/node_modules/@mantine/core/styles.css b/node_modules/@mantine/core/styles.css
+index d440fbf..dc78ac1 100644
+--- a/node_modules/@mantine/core/styles.css
++++ b/node_modules/@mantine/core/styles.css
+@@ -1,34 +1,4 @@
+-/* ----- CSS reset ----- */
+-*,
+-*::before,
+-*::after {
+-  box-sizing: border-box;
+-}
+-
+-input,
+-button,
+-textarea,
+-select {
+-  font: inherit;
+-}
+ 
+-button,
+-select {
+-  text-transform: none;
+-}
+-
+-/* ----- Global styles ----- */
+-body {
+-  margin: 0;
+-  font-family: var(--mantine-font-family);
+-  font-size: var(--mantine-font-size-md);
+-  line-height: var(--mantine-line-height);
+-  background-color: var(--mantine-color-body);
+-  color: var(--mantine-color-text);
+-
+-  -webkit-font-smoothing: var(--mantine-webkit-font-smoothing);
+-  -moz-osx-font-smoothing: var(--mantine-moz-font-smoothing);
+-}
```

Comment:
This PR seems to also delete the CSS resets and global styles, is this intentional?

Reply:
That was already there in the previous patch (it's something they did for the sdk, as otherwise we'd leak those style outside).
Git managed to show it as a renamed file when i did the first commit but decided it was a delete + add when i pushed the second commit

Reply:
oh, my bad! all good then